docs/cluster-configuration.md

```diff
@@ -750,6 +750,23 @@ Please correctly consider the implications of this option before setting it.
 
 `BlockUninstallIfWorkloadsExist` is the default behaviour.
 
+
+## Cluster level EvictionStrategy
+
+`UninstallStrategy` defines how to proceed on uninstall when workloads (VirtualMachines, DataVolumes) still exist:
+
+`EvictionStrategy` defines at the cluster level if the VirtualMachineInstance should be
+migrated instead of shut-off in case of a node drain. If the VirtualMachineInstance specific
+field is set it overrides the cluster level one.
+Possible values:
+
+- `EvictionStrategyNone` no eviction strategy at cluster level.
+- `EvictionStrategyLiveMigrate` ???
+- `EvictionStrategyExternal` ???
```

Comment:
Can you please document the behaviour here?

Reply:
done, thank you.